src/Microsoft.AspNetCore.Razor/CodeGenerators/GeneratedTagHelperContext.cs

```diff
@@ -209,10 +209,10 @@ public GeneratedTagHelperContext()
         public string TagHelperOutputContentPropertyName { get; set; }
 
         /// <summary>
-        /// The name of the method on the property <see cref="ExecutionContextOutputPropertyName"/> used to retrieve
-        /// tag helper child content.
+        /// The name of the method on the property <see cref="ExecutionContextOutputPropertyName"/> used to execute
+        /// child content and set the rendered results on its <see cref="ExecutionContextOutputPropertyName"/> property.
         /// </summary>
-        public string TagHelperOutputGetChildContentAsyncMethodName { get; set; }
+        public string ExecutionContextSetOutputContentAsyncMethodName { get; set; }
```

Comment:
Cool ... at least I spotted this one before my compile resulted in a surprising :boom:. :smile: